common.props

```diff
@@ -0,0 +1,38 @@
+<Project>
+  
+  <PropertyGroup>
+    <BuildVersion>3.4.0</BuildVersion>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyOriginatorKeyFile>../../buildscripts/CastleKey.snk</AssemblyOriginatorKeyFile> 
+    <SignAssembly>true</SignAssembly> 
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Authors>Castle Project Contributors</Authors>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/castleproject/Windsor</RepositoryUrl>
+    <PackageLicenseUrl>http://www.apache.org/licenses/LICENSE-2.0.html</PackageLicenseUrl>
+    <PackageProjectUrl>http://www.castleproject.org/projects/windsor/</PackageProjectUrl>
+    <PackageIconUrl>http://www.castleproject.org/img/windsor-logo.png</PackageIconUrl>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageOutputPath>../../Packages/</PackageOutputPath>
+    <PackageVersion>$(BuildVersion)</PackageVersion>
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSource>false</IncludeSource>
+    <Copyright>Copyright 2004 Castle Project - http://www.castleproject.org/ </Copyright>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);DOTNET45</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\BreakingChanges.txt" />
```

Comment:
I see the packages contain a "content" and a "contentFiles" folder both with this set of files. How does this work, what do these directories do because we used to have these files in the root of the package?

If VS doesn't automatically open readme.txt we can get rid of that one.